cli/dcoscli/marathon/main.py

```diff
@@ -860,6 +865,32 @@ def task_stop(self, task_id, wipe):
         emitter.publish(task)
         return 0
 
+    def task_kill(self, task_ids, scale, wipe):
+        """Kill one or multiple Marathon tasks
+
+        :param task_ids: the id of the task
+        :type task_ids: [str]
+        :param scale: Scale the app down after killing the specified tasks
+        :type scale: bool
+        :param wipe: whether remove reservations and persistent volumes.
+        :type wipe: bool
+        :returns: process return code
+        :rtype: int
+        """
+
+        if not task_ids:
+            raise DCOSException('Please provide at least one task ID')
+
+        client = self._create_marathon_client()
+        payload = client.kill_and_scale_tasks(task_ids, scale, wipe)
+
+        emitter.publish(payload)
+        # No deployment was started and no tasks were killed
```

Comment:
If we get to that part of the code, it means marathon returned a status 200. If there was an error, wouldn't we get a different status code? 

If the taskId is unknown, wouldn't that return a different status code and thus failure earlier in the code?